search-relevancy-test/src/search_relevancy_test/anomaly_analyzer.clj

```diff
@@ -11,8 +11,8 @@
   "Get the test data from the CSV file - need the concept ids we want to look at
   and the search params. If the anomaly has multiple tests, we get the data for
   all of the tests."
-  [anomaly-number]
-  (let [csv-data (core/read-anomaly-test-csv)
+  [anomaly-number file-name]
```

Comment:
This is why every project should have tests. I didn't realize I broke this when adding the EDSC metrics based anomalies.

Reply:
Yeah. Maybe we could address the testing issue after moving it to its own repository.